packages/amplication-client/src/Resource/EntitiesTile.tsx

```diff
@@ -23,6 +23,7 @@ const EVENT_DATA: TrackEvent = {
 
 function EntitiesTile({ resourceId }: Props) {
   const history = useHistory();
+  const {workspace,project} = useParams();
```

Comment:
did you run formatter? (It seems like you are missing spaces..)

Reply:
@yuval-hazaz  yes, the CI pass .. i will check again just to be sure

Reply:
@mshidlov, something looks strange in the CI... I would expect the CI to fail on formatting error